lib/piv/views/piv_screen.dart

```diff
@@ -57,6 +58,14 @@ class _PivScreenState extends ConsumerState<PivScreen> {
   Widget build(BuildContext context) {
     final l10n = AppLocalizations.of(context)!;
     final hasFeature = ref.watch(featureProvider);
+    final (fipsCapable, fipsApproved) = ref
+            .watch(currentDeviceDataProvider)
+            .valueOrNull
+            ?.info
+            .getFipsStatus(Capability.piv) ??
+        (false, false);
```

Comment:
this could be a turned into provider on its own and reused in several places making the code shorter

Reply:
I think it's a balance between repeating some code, and having too many providers with very low usage. If this pattern is repeated too much it makes sense to make it a Provider of its own, but as it is now I don't really think it's warranted.